docker_sigprofiler/command_line.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+from sigproextractor import sigpro as sig
```

Comment:
seems not necessary to use `as`. We usually use it to avoid namespace clashes or make some imported names more readable in the code. Neither seems the case in this file.